app/tests/helpers/mockData.ts

```diff
@@ -0,0 +1,62 @@
+// Note: This file is loaded by Storybook.
+// Don't directly call jest in it.
```

Comment:
nit: could put this in something like an `/app/lib/` folder or something to make it clearer it's not solely a test helper as well